README.md

```diff
@@ -1,10 +1,10 @@
 # TM4J Jenkins Plugin
 
-A plugin for Jenkins built for integrating automated tests with [Test Management for Jira Server](https://marketplace.atlassian.com/apps/1213259/test-management-for-jira?hosting=server&tab=overview) (TM4J).  
+A plugin for Jenkins built for integrating automated tests with [Test Management for Jira Server](https://marketplace.atlassian.com/apps/1213259/test-management-for-jira?hosting=server&tab=overview) and [Test Management for Jira Cloud](https://marketplace.atlassian.com/apps/1213259/tm4j-test-management-for-jira?tab=overview&hosting=cloud) (TM4J).  
 
 The plugin introduces two new configuration steps for Jenkins jobs:
-1) A build step to export Gherkin feature files (BDD) from TM4J
-1) A post-build step to import the test results back to TM4J
+1) A build step to import Gherkin feature files (BDD) from TM4J (currently available only for Test Management for Jira Server)
+1) A post-build step to export the test results back to TM4J
```

Comment:
What if we use the same naming convention as we have for the Build Steps title: "download Gherkin..." and "publish test results..."